roles/provision-metrics-apb/files/mobile-services-dashboard.json

```diff
@@ -648,7 +650,7 @@
           "tableColumn": "",
           "targets": [
             {
-              "expr": "sum(nodejs_heap_size_total_bytes + nodejs_heap_size_used_bytes)",
+              "expr": "sum(process_resident_memory_bytes{service=\"data-sync-server\"})",
```

Comment:
For reference, resident memory (RSS) is the closest information available when the mem usage is reported inside the container and inside the process. See https://nodejs.org/api/process.html#process_process_memoryusage

RSS is basically how much mem is used on physical memory by the process. Not the mem on disk when page swaps occur.

I checked how the mem usage is collected by Nagios as I was told that's pretty stable.

It is here: https://github.com/feedhenry/nagios-container/blob/master/plugins/default/lib/memory_usage.py#L31

In that case, Crgoups is used and `/sys/fs/cgroup/memory/memory.usage_in_bytes` reports the memory usage of the entire system (in our case, pod). This is 100% accurate with `docker stats`.

However, we didn't want to go this way and implement a custom metric gauge because the `process_resident_memory_bytes` metric gauge provided by Prometheus node client is accurate enough.

Value reported by `process_resident_memory_bytes` simply only lacks the overhead: 

- Docker overhead
- OS overhead